src/main/resources/resources/logisim/strings/fpga/fpga.properties

```diff
@@ -375,4 +376,3 @@ softwaresQuestaPathLabel = Questa Advanced Simulator path:
 softwaresQuestaValidationLabel = Use Questa Advanced Simulator to validate HDL entities
 softwaresTitle = Software
 VivadoToolPath = Xilinx Vivado toolpath:
-
```

Comment:
Does the previous line still finish with an end of line character?

Reply:
No. It should be explicit `\n` for that. Also it looks `\n` should be be here as this is a field label string.